### ISSUE TITLE
Questions about training dataset construction

### ISSUE THREAD

**Issue by Kanraaaaa** [Leezp99/LongContext_CAM_TTS#1]
Hi, Zhipeng.
Thank you for sharing your demo of CAM-based long-context TTS. I have read your paper and I have some questions about training dataset construction as follows:
Do the CAM-blocks train together with the LM？
    If yes,  how can I get Mem_n-1 for each Mem_n？ For a single training sample, I can only obtain the text, audio, and a few preceding sentences of text and audio.
    If no, we can get  Mem_n by teacher-forcing, but what loss function can we use to train CAM-blocks?
